cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1190,4 +1194,47 @@ protected void addContainerEnvsToExistingEnvs(List<EnvVar> existingEnvs, List<Co
             }
         }
     }
+
+    protected void javaOptionsForEO(List<EnvVar> envVars) {
```

Comment:
If it is just for EO, should it be maybe in some EO class and not in the Abstratc Model which is shared for everything?